overlord/snapstate/aux_store_info_test.go

```diff
@@ -0,0 +1,63 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2018 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package snapstate_test
+
+import (
+	"gopkg.in/check.v1"
+
+	"github.com/snapcore/snapd/dirs"
+	"github.com/snapcore/snapd/osutil"
+	"github.com/snapcore/snapd/overlord/snapstate"
+	"github.com/snapcore/snapd/snap"
+)
+
+type auxInfoSuite struct{}
+
+var _ = check.Suite(&auxInfoSuite{})
+
+func (s *auxInfoSuite) SetUpTest(c *check.C) {
+	dirs.SetRootDir(c.MkDir())
+}
+
+func (s *auxInfoSuite) TestAuxStoreInfoRoundTrip(c *check.C) {
+	media := snap.MediaInfos{{Type: "1-2-3-testing"}}
+	info := &snap.Info{SuggestedName: "some-snap"}
+	info.SnapID = "some-id"
+	filename := snapstate.AuxStoreInfoFilename(info.SnapID)
```

Comment:
Would it make sense to add a test here that checks that filename is of the expected form? Something like `c.Check(filename, Equals, dirs.StoreAuxDir+"some-id.json")` ?

Reply:
I'll add a unit test for `AuxStoreInfoFilename`.